### PR TITLE
Run black and isort over the codebase.

### PR DESCRIPTION
**Pull request by tomprince**
The only manual source changes are to `src/_zkapauthorizer/api.py`, which is in the final commit.

I will follow up with a commit that runs flake8, since that requires manual changes.

### CODE REVIEW

File: src/_zkapauthorizer/tests/test_storage_protocol.py [hunk 1]
```diff
@@ -16,128 +16,79 @@
...
 )
-from testtools.twistedsupport import (
-    succeeded,
-    failed,
-)
-from testtools.twistedsupport._deferred import (
-    # I'd rather use https://twistedmatrix.com/trac/ticket/8900 but efforts
-    # there appear to have stalled.
+from testtools.twistedsupport import failed, succeeded
+from testtools.twistedsupport._deferred import (  # I'd rather use https://twistedmatrix.com/trac/ticket/8900 but efforts; there appear to have stalled.
```
Comment (exarkun) [inline src/_zkapauthorizer/tests/test_storage_protocol.py:39]: will black let us put this comment on the line above instead, at least?

File: src/_zkapauthorizer/tests/test_storage_protocol.py [hunk 2]
```diff
@@ -16,128 +16,79 @@
...
+    cleanup_storage_server,
+    get_passes,
+    pass_factory,
+    privacypass_passes,
+    whitebox_write_sparse_share,
+    write_toy_shares,
 )
-from ..foolscap import (
-    ShareStat,
+from .strategies import (  # Not really a strategy...
```
Comment (exarkun) [inline src/_zkapauthorizer/tests/test_storage_protocol.py:72]: arguably the meaning of this comment has been lost by the change

**Review by exarkun — APPROVED** [review]
Thanks.  I had some WTF moments reading the new formatting.  However, hooray consistency and tool enforcement all that.

There are a couple changes to comment formatting that I hope can be fixed but otherwise this seems landable.

Please fix the points noted inline to the extent possible and then merge.

File: src/_zkapauthorizer/tests/test_storage_protocol.py (hunk 1 above)
Comment (tomprince) [reply src/_zkapauthorizer/tests/test_storage_protocol.py:39]: I've made this change in the flake8 PR. I'll note that it is actually isort that makes this change; I searched for a while to see if there was a way to disable moving comments, unsuccessfully.

File: src/_zkapauthorizer/_version.py [hunk 3]
```diff
@@ -1,4 +1,3 @@
-
```
[the remark this answers is not in the document]
Comment (tomprince) [reply src/_zkapauthorizer/_version.py:1]: I've updated the configuration to skip this file (and reverted the change to it).

File: src/_zkapauthorizer/tests/test_storage_protocol.py (hunk 1 above)
Comment (tomprince) [reply src/_zkapauthorizer/tests/test_storage_protocol.py:39]: (I've actually moved it above in this PR now).

File: src/_zkapauthorizer/tests/test_storage_protocol.py (hunk 2 above)
Comment (tomprince) [reply src/_zkapauthorizer/tests/test_storage_protocol.py:72]: After some experimentation, I've found that trailing comments on an import line are preserved, so I've moved this to the appropriate place. I also filed https://github.com/PyCQA/isort/issues/1843 to see if this behavior can be better documented.

**Pull request merged**